### PR TITLE
Fix gcc9 warnings in CondTools & CondFormats

### PR DESCRIPTION
**Pull request by mrodozov**
#### PR description:

Fix gcc9 warnings

#### PR validation:

Builds without warnings

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: CondTools/HLT/src/AlCaRecoTriggerBitsRcdRead.cc [hunk 1]
```diff
@@ -147,6 +147,7 @@ void AlCaRecoTriggerBitsRcdRead::printMap(edm::RunNumber_t firstRun,
     case kPython:
       output << "  triggerLists = cms.VPSet(\n";
       // no 'break;'!
```
[the remark this answers is not in the document]
Comment (mrodozov) [reply CondTools/HLT/src/AlCaRecoTriggerBitsRcdRead.cc]: alright I'll remove them. I might remove also the non used cases but only after an agreement

**Comment by Martin-Grunewald** [comment]
+1

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
sry about that I had to remove a comment as agreed

**Comment by pohsun** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Comment by ggovi** [comment]
+1

**Pull request merged**